### DISCUSSION TITLE
UMEP Required Libraries offline installation

### DISCUSSION THREAD

**Discussion by atharvauk110401** [UMEP-dev/UMEP#451] [General]
Please mention how to install UMEP required libraries in offline or list the required libraries for UMEP, as our device cannot access internet due to some security and privacy policy.

**Comment by biglimp** [comment]
QGIS changed operational Python version as from 3.18, so if you have an earlier QGIS version e.g. 3.16, than you will not be able to use UMEP properly. I think you might be able to install libraries using the OSGeo shell and the pip command (https://umep-docs.readthedocs.io/en/latest/Getting_Started.html#adding-missing-python-libraries-and-other-osgeo-functionalities) without being administrator of your computer. I cannot provide all libraries that are required cos I dont know them all.
